pkg/cloudprovider/cloudapi/azure/azure_compute.go

```diff
@@ -91,6 +92,7 @@ func newComputeServiceConfig(account types.NamespacedName, service azureServiceC
 		inventoryStats:         &internal.CloudServiceStats{},
 		credentials:            credentials,
 		computeFilters:         make(map[string][]*string),
+		selectors:              make(map[string]*crdv1alpha1.CloudEntitySelector),
```

Comment:
Can you add a comment here explaining why you need to selector for Azure?

Reply:
done